### ISSUE TITLE
Multiple <template> using

### ISSUE THREAD

**Issue by takiguru** [Akryum/vue-meteor#167]
Hi! When i want to use "Scoped Slots" which require multiple <template> keyword i'm running into compile problems. 

**Log:**
app/pages/Users.vue:13: Expected one of: <template>, <script>, <style>

**Version:**
akryum:vue@1.2.2
akryum:vue-component@0.8.18
akryum:vue-component-dev-client@0.2.9
akryum:vue-component-dev-server@0.0.7

I want to use Element UI components for Front End , but they are using this method a lot. So is there any way to use multiple <template> in the component?

**Comment by thearabbit** [comment]
+1, I have the same problem for using `Custom Table Culumn`
```js
<el-table-column
      label="Operations">
      <template scope="scope">
        <el-button
          size="small"
          @click="handleEdit(scope.$index, scope.row)">Edit</el-button>
        <el-button
          size="small"
          type="danger"
          @click="handleDelete(scope.$index, scope.row)">Delete</el-button>
      </template>
    </el-table-column>
```

**Comment by Akryum** [comment]
Fixed in `0.9.0` release.

**Comment by thearabbit** [comment]
I will try soon.
💯
